README.md

```diff
@@ -1 +1,289 @@
-# Sprinkler
+<div align="center">
+
+![Sprinkler logo](img/Sprinkler_light.svg#gh-light-mode-only)
+![Sprinkler logo](img/Sprinkler_dark.svg#gh-dark-mode-only)
+
+<h3><i>A non-intrusive set of extensions sprinkled over the Kotlin standard library.</i></h3>
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.black-kamelia/sprinkler.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.black-kamelia%22%20AND%20a:%22sprinkler%22)
+[![Build status](https://ci.black-kamelia.com/buildStatus/icon?subject=Build&job=Sprinkler%2Fmaster)](https://ci.black-kamelia.com/job/Sprinkler/)
+</div>
+
+## 📋 Summary
+
+- [📋 Summary](#-summary)
+- [🤔 What is it?](#-what-is-it)
+- [✨ Modules and Documentation](#-modules-and-documentation)
+- [🛠 Integration](#-integration)
+- [📝 License](#-license)
+
+## 🤔 What is it?
+
+**Sprinkler** is an open-source set of extensions to the **[Kotlin](https://kotlinlang.org/)** standard library, in the
+same vein as projects such as ***kotlinx***.
+
+## ✨ Modules and Documentation
+
+- ***Collections***: cast-safe read-only collections for Kotlin (see [documentation](docs/collections.md))
```

Comment:
It may be better to place the link directly on the module's name to avoid redundancy ('see doc' for each module in the list)